Content.Server/MachineLinking/Components/ReleaseSignallerHolderComponent.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Content.Server.MachineLinking.Components;
+
+/// <summary>
+/// Added to people holding a release signaller for detonating when dropped..
+/// </summary>
+[RegisterComponent]
+public sealed class ReleaseSignallerHolderComponent : Component
+{
+    public HashSet<ReleaseSignallerComponent> Switches = new();
```

Comment:
Should use EntityUids because these aren't serializable, also VV, also you don't seem to handle these being shutdown?